e2e/pageobjects/ide/ProjectTree.ts

```diff
@@ -40,13 +40,14 @@ export class ProjectTree {
     }
 
     async waitItemExpanded(itemPath: string, timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
-        const expandedItemLocator: By = By.css(this.getExpandedItemCssLocator(itemPath));
-
+        const locator: string = await this.getExpandedItemCssLocator(itemPath);
```

Comment:
Could you explain all your changes here? What do you do, what purpose of this and what you expected.

Reply:
We implement changes according to separated issue: https://github.com/eclipse/che/issues/14297